src/panels/lovelace/components/hui-input-list-editor.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  html,
+  LitElement,
+  PropertyDeclarations,
+  TemplateResult,
+} from "lit-element";
+import "@polymer/paper-input/paper-input";
+
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
+
+import { EditorTarget } from "../editor/types";
+
+export class HuiInputListEditor extends LitElement {
+  protected value?: string[];
+  protected hass?: HomeAssistant;
+  protected heading?: string;
+  protected inputLabel?: string;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      value: {},
+      heading: {},
+      inputLabel: {},
+    };
+  }
+
+  protected render(): TemplateResult | void {
+    if (!this.value) {
+      return html``;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <h3>${this.heading}</h3>
```

Comment:
This component shouldn't have its own heading, that reduces reusability.

Reply:
OK. I moved the heading to the map card editor